### ISSUE TITLE
Gem fails to build on 10.11 beta 6.

### ISSUE THREAD

**Issue by fjwagner** [AXElements/accessibility_core#17]
When I try to gem install accessibility_core on MAC OS X 10.11 beta 6, the install fails because of make failures.  There are a some constants that are going away for error enumeration in 10.11.  There are some deprecated functions in 10.11 as well.

Details:
When trying build accessibility_core-0.6.1/ext/accessibility/bridge
running 'ruby extconf.rb; make'
I get:
compiling bridge.c
bridge.c:178:10: error: expression is not an integer constant expression; folding it to a constant is a GNU extension [-Werror,-Wgnu-folding-constant]
    case kAXValueIllegalType:
         ^~~~~~~~~~~~~~~~~~~
bridge.c:180:10: error: expression is not an integer constant expression; folding it to a constant is a GNU extension [-Werror,-Wgnu-folding-constant]
    case kAXValueCGPointType:
         ^~~~~~~~~~~~~~~~~~~
bridge.c:182:10: error: expression is not an integer constant expression; folding it to a constant is a GNU extension [-Werror,-Wgnu-folding-constant]
    case kAXValueCGSizeType:
         ^~~~~~~~~~~~~~~~~~
bridge.c:184:10: error: expression is not an integer constant expression; folding it to a constant is a GNU extension [-Werror,-Wgnu-folding-constant]
    case kAXValueCGRectType:
         ^~~~~~~~~~~~~~~~~~
bridge.c:186:10: error: expression is not an integer constant expression; folding it to a constant is a GNU extension [-Werror,-Wgnu-folding-constant]
    case kAXValueCFRangeType:
         ^~~~~~~~~~~~~~~~~~~
bridge.c:188:10: error: expression is not an integer constant expression; folding it to a constant is a GNU extension [-Werror,-Wgnu-folding-constant]
    case kAXValueAXErrorType:
         ^~~~~~~~~~~~~~~~~~~
6 errors generated.
make: *** [bridge.o] Error 1

When trying build accessibility_core-0.6.1/ext/accessibility/extras
running 'ruby extconf.rb; make'
I get:
compiling extras.c
extras.c:789:66: error: 'NSWorkspaceLaunchAllowingClassicStartup' is deprecated: first deprecated in OS X 10.11 - The Classic environment is no longer supported. [-Werror,-Wdeprecated-declarations]
    WORKSPACE_CONST("NSWorkspaceLaunchAllowingClassicStartup",   NSWorkspaceLaunchAllowingClassicStartup);
                                                                 ^
extras.c:782:79: note: expanded from macro 'WORKSPACE_CONST'
#define WORKSPACE_CONST(str, val) rb_define_const(rb_cWorkspace, str, INT2FIX(val));
                                                                              ^
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/ruby/ruby.h:241:45: note: expanded from macro 'INT2FIX'
#define INT2FIX(i) ((VALUE)(((SIGNED_VALUE)(i))<<1 | FIXNUM_FLAG))
                                            ^
/System/Library/Frameworks/AppKit.framework/Headers/NSWorkspace.h:32:5: note: 'NSWorkspaceLaunchAllowingClassicStartup' has been explicitly marked deprecated here
    NSWorkspaceLaunchAllowingClassicStartup   NS_ENUM_DEPRECATED_MAC(10_3, 10_11, "The Classic environment is no longer supported.") = 0x00020000,
    ^
extras.c:790:66: error: 'NSWorkspaceLaunchPreferringClassic' is deprecated: first deprecated in OS X 10.11 - The Classic environment is no longer supported. [-Werror,-Wdeprecated-declarations]
    WORKSPACE_CONST("NSWorkspaceLaunchPreferringClassic",        NSWorkspaceLaunchPreferringClassic);
                                                                 ^
extras.c:782:79: note: expanded from macro 'WORKSPACE_CONST'
#define WORKSPACE_CONST(str, val) rb_define_const(rb_cWorkspace, str, INT2FIX(val));
                                                                              ^
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/include/ruby-2.0.0/ruby/ruby.h:241:45: note: expanded from macro 'INT2FIX'
#define INT2FIX(i) ((VALUE)(((SIGNED_VALUE)(i))<<1 | FIXNUM_FLAG))
                                            ^
/System/Library/Frameworks/AppKit.framework/Headers/NSWorkspace.h:33:5: note: 'NSWorkspaceLaunchPreferringClassic' has been explicitly marked deprecated here
    NSWorkspaceLaunchPreferringClassic        NS_ENUM_DEPRECATED_MAC(10_3, 10_11, "The Classic environment is no longer supported.") = 0x00040000,
    ^
2 errors generated.
make: *** [extras.o] Error 1

**Comment by fjwagner** [comment]
I created a pull request that will fix this issue.

**Comment by fjwagner** [comment]
Fixed with https://github.com/AXElements/accessibility_core/releases/tag/v0.6.2
